### PR TITLE
Fix Postgresql resource type name

### PR DESCRIPTION
**Pull request by maxpoullain**
I updated the resource type name for the PostgreSQL detectors.
It was broken and not working anymore.

### CODE REVIEW

**Review by xp-1000 — APPROVED** [review]

**Comment by Shr3ps** [comment]
Since when this is broken? On which tenant did you test?
Official doc says: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdbforpostgresqlservers

**Comment by maxpoullain** [comment]
> Since when this is broken? On which tenant did you test?
> Official doc says: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdbforpostgresqlservers

No idea since when this is broken. It's my first time implementing it.

What I'm sure though, it's that I was correctly getting metrics with the Azure integrations but no data was found on the detectors themselves.
Upon further inspection of the Signalflow, this is the difference I found: 
Metrics are getting scrapped from the resource type name `Microsoft.DBForPostgreSQL/servers` from the Azure integration but on the PostgreSQL detector, the resource type name is `Microsoft.DBforPostgreSQL/servers`, with no data getting reported.
I tried my fix and it's working fine now.

**Comment by maxpoullain** [comment]
Updated with `Microsoft.DB*orPostgreSQL/servers` to match both PostgreSQL resource type name.

**Comment by xp-1000** [comment]
hello @maxpoullain the readme order is not right, may be you updated it manually instead of using the makefile ?

**Comment by xp-1000** [comment]
@Shr3ps is it ok for you now ?

File: modules/integration_azure-postgresql/detectors-postgresql.tf [hunk 1]
```diff
@@ -33,7 +33,7 @@ resource "signalfx_detector" "cpu_usage" {
   tags                    = compact(concat(local.common_tags, local.tags, var.extra_tags))
 
   program_text = <<-EOF
-        base_filter = filter('resource_type', 'Microsoft.DBforPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
+        base_filter = filter('resource_type', 'Microsoft.DB?orPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
```
Comment (BzSpi) [inline modules/integration_azure-postgresql/detectors-postgresql.tf:36]:
```suggestion
        base_filter = filter('resource_type', 'Microsoft.DBForPostgreSQL/servers', 'Microsoft.DBforPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
```
More readable in my opinion

**Review by Shr3ps — APPROVED** [review]

**Pull request merged**